src/wp-content/themes/twentynineteen/style-rtl.css

```diff
@@ -3006,7 +3006,7 @@ body.page .main-navigation {
   margin-left: calc( .25 * 1rem);
 }
 
-.main-navigation .main-menu .menu-item-has-children:not(.off-canvas)[focus-within] > .sub-menu {
+.main-navigation.js-focus-within .main-menu .menu-item-has-children:not(.off-canvas)[focus-within] > .sub-menu, .js-focus-within .main-navigation .main-menu .menu-item-has-children:not(.off-canvas)[focus-within] > .sub-menu {
```

Comment:
The specificity here is changing from `0,0,6,0` to `0,0,7,0` and the selector is changed from one to two.

Has the default config of focus-within changed as the `js-focus-within` class is new?

Reply:
So I tracked this down. This appears to be something introduced within the [6.0.0 update ](https://github.com/csstools/postcss-plugins/blob/main/plugins/postcss-focus-within/CHANGELOG.md#600-july-8-2022)of `postcss-focus-within`. It looks like version[ 6.1.0 added the ability to prevent this from being added](https://github.com/csstools/postcss-plugins/blob/main/plugins/postcss-focus-within/CHANGELOG.md#610-july-30-2022).

I've gone and added that to the PostCSS configs. But now there's a different issue where the same selectors are added twice.

Reply:
Turns out the duplicate classes were a bug in PostCSS and has been resolved in the latest version of `postcss-focus-within` (see csstools/postcss-plugins#835 and csstools/postcss-plugins#840).

After updating, these issues have all been resolved.